tests/test_check_chronos_jobs.py

```diff
@@ -137,44 +137,68 @@ def test_sensu_event_for_last_run_state_invalid():
 
 
 @patch('paasta_tools.check_chronos_jobs.chronos_tools.lookup_chronos_jobs', autospec=True)
+def test_respect_latest_run_after_rerun(mock_lookup_chronos_jobs):
+    fake_job = {
+        'name': 'service1 test-job',
+        'lastSuccess': '2016-07-26T22:00:00+00:00',
+        'lastError': '2016-07-26T22:01:00+00:00'
+    }
+    mock_lookup_chronos_jobs.side_effect = iter([[
+        fake_job
+    ]])
+
+    fake_configured_jobs = [('service1', 'chronos_job')]
+    fake_client = Mock(list=Mock(return_value=[('service1', 'chronos_job')]))
+
+    assert check_chronos_jobs.build_service_job_mapping(fake_client, fake_configured_jobs) == {
+        ('service1', 'chronos_job'): [(fake_job, chronos_tools.LastRunState.Fail)]
+    }
+
+    # simulate a re-run where we now pass
+    reran_job = {
+        'name': chronos_tools.TMP_JOB_IDENTIFIER + ' ' + fake_job['name'],
```

Comment:
Is it possible to use the exact same function that re_run uses to construct this job name?